packages/deck-scripts/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@spinnaker/deck-scripts",
```

Comment:
I'm not loving this name.  all npm pckages in`@spinnaker/` are "deck" and scripts doesn't fully describe what this package does.  Maybe `@spinnaker/build` or `@spinnaker/bundle`.

On second thought, `deck-scripts` does mimic `react-scripts` so maybe `@spinnaker/scripts`?